feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableSheet.kt

```diff
@@ -128,17 +127,11 @@ fun TimetableSheet(
                 }
 
                 is GridTimetable -> {
-                    val nestedScrollDispatcher = remember { NestedScrollDispatcher() }
```

Comment:
Thanks for the great PR!
Is there any reason why you originally applied the NestedScrollDispatcher definition and Modifier nestedScroll here?

Reply:
At first I was misunderstanding that I needed to pass a nestedScrollDispatcher and a timetableSheetContentScrollState.nestedScrollConnection to one Modifier.nestedScroll() argument. 
But in this case, the TimetableGrid's nestedScrollDispatcher is independent of the TimetableSheet. Therefore I moved the dispatcher into TimetableGrid class.